Fix and improve SDK 6 doc about tests (#722)
MicroEJ/docs: SDK6UserGuide/testProject.rst

Refer to the Gradle `Test filtering <https://docs.gradle.org/current/userguide/java_testing.html#test_filtering>`__
documentation for more details on how to filter the tests and on the available patterns.

.. warning::

   At the moment, only class-level filtering is supported. 

>    This means that, for instance, you are unable to run a single test method within a test class.

Reviewer: ```suggestion
   This means that, for instance, it is not possible to run a single test method within a test class.
```

Author: So you don't like what ChatGPT suggests? ;) It is because of the usage of "you"?

Reviewer: the `you` thing, yes. Not a formal rule, I know.

Author: Applied. Not sure too.
